discord/enums.py

```diff
@@ -658,7 +658,7 @@ class SlashCommandOptionType(Enum):
 
     @classmethod
     def from_datatype(cls, datatype):
-        if isinstance(datatype, tuple):  # typing.Union has been used
+        if isinstance(datatype, (tuple, list)):  # typing.Union has been used
```

Comment:
I don't think this should be changed, the comment states that Union is converted to a tuple.
```suggestion
        if isinstance(datatype, tuple):  # typing.Union has been used
```

Reply:
This still causes the error from #991 unless you change it to include both.